app/src/main/res/layout/profile_chooser_profile_view.xml

```diff
@@ -79,6 +80,7 @@
 
         <TextView
           android:id="@+id/profile_is_admin_text"
+          style="@style/TextViewCenterHorizontal"
```

Comment:
Isn't this conditional? Ditto for profile_last_visited.

Reply:
How to set conditional style for the views? I found that is is not doable https://code.google.com/archive/p/android-developer-preview/issues/2613.

Reply:
I created new style called `SubtitleLight` that matches the Textviews property without having gravity attribute.

Reply:
The style thing is a bit of a conundrum. I think defaulting the property is probably the best that we can do, but it's probably going to end up causing some confusion when we figure out the long-term styles & add the CI check. Can you add a comment here & to other versions of this text view (ie in other versions of this file) to explain why this style is used vs. one that defines the text alignment?

Reply:
Added comment please check if that sounds good.